tests/unit/RegistryHandler.ts

```diff
@@ -64,7 +66,7 @@ registerSuite({
 		assert.isTrue(registryHandler.has(bar));
 	},
 	'get'() {
-		const promise = new Promise((resolve) => {
+		const promise = new Promise<WidgetBaseConstructor>((resolve) => {
```

Comment:
_sigh_ I thought I had the typings right with 2.4.1 for that to work without needing to cast...